### PR TITLE
Associated facilities, patients, and visits with users for debugging

### PR DESCRIPTION
**Pull request by timcheadle**
We would like to know who originally authored each model to make debugging easier.

### CODE REVIEW

File: app/models/facility.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 class Facility < ApplicationRecord
-  belongs_to :district
+  belongs_to :district, inverse_of: :facilities
+  belongs_to :user
```
Comment (govindkrjoshi) [inline app/models/facility.rb]:
```suggestion
  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

`belongs_to :user` threw me off.

File: app/models/patient.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@ class Patient < ApplicationRecord
   include SyncLoggable
 
   belongs_to :facility, inverse_of: :patients
+  belongs_to :user
```
Comment (govindkrjoshi) [inline app/models/patient.rb]:
```suggestion
  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

File: app/models/visit.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 class Visit < ApplicationRecord
   include SyncLoggable
   belongs_to :patient, inverse_of: :visits
+  belongs_to :user
```
Comment (govindkrjoshi) [inline app/models/visit.rb]:
```suggestion
  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

File: db/migrate/20181206030107_add_user_id_to_cards.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+class AddUserIdToCards < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :facilities, :user, index: true
```
Comment (govindkrjoshi) [inline db/migrate/20181206030107_add_user_id_to_cards.rb]:
```suggestion
    add_column :facilities, :author_id, :int, index: true
```

File: db/migrate/20181206030107_add_user_id_to_cards.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+    add_reference :patients, :user, index: true
```
Comment (govindkrjoshi) [inline db/migrate/20181206030107_add_user_id_to_cards.rb]:
```suggestion
    add_column :facilities, :author_id, :int, index: true
```

File: db/migrate/20181206030107_add_user_id_to_cards.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+    add_reference :visits, :user, index: true
```
Comment (govindkrjoshi) [inline db/migrate/20181206030107_add_user_id_to_cards.rb]:
```suggestion
    add_column :facilities, :author_id, :int, index: true
```

File: db/migrate/20181206030107_add_user_id_to_cards.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+  end
```
Comment (govindkrjoshi) [inline db/migrate/20181206030107_add_user_id_to_cards.rb]:
```suggestion
  end
  
  add_foreign_key :facilities, :users, column: :author_id
  add_foreign_key :patients, :users, column: :author_id
  add_foreign_key :visits, :users, column: :author_id
```

**Comment by timcheadle** [comment]
@govindkrjoshi  you are totally right; `author` makes much more sense. I've made all of your changes (and some you didn't catch 😉 )

**Pull request merged**